### DISCUSSION TITLE
How to enable in incognito mode

### DISCUSSION THREAD

**Discussion by vfd65536** [ScratchAddons/ScratchAddons#1113] [Help] [answered]
Unlike my other chrome extensions, this extension does not seem to have the option to enable it in Incognito Mode, as shown below:
![image](https://user-images.githubusercontent.com/76582805/103189592-5e0d6900-489b-11eb-9164-d54141007ab1.png)
Is there any way to enable this extension in Incognito Mode, or am I forced to use this extension only in the standard browser?

**Comment by mxmou** [comment, marked as answer]
Scratch Addons doesn't work in incognito mode, see #203.

**Reply by Marc92020** [reply]
Yeah it doesn't so I think this is the answer

**Reply by WorldLanguages** [reply]
Yeah, unfortunately because of Firefox we can't provide incognito support to Chrome users.
@vfd65536 If this is a really big deal for you, feel free to fork the extension, and change `not_allowed` to `split` in manifest.json and it should work just fine on Chrome.
